### DISCUSSION TITLE
Can Mergify set labels on PRs that update mergify.yml?

### DISCUSSION THREAD

**Discussion by nixpanic** [Mergifyio/mergify#2546] [Workflows]
When Mergify detects changes to `.mergify.yml` it will not merge PRs automatically, but maintainers of the repository need to do this.

Unfortunately, it also seems that labels are not set on the PR where `.mergify.yml` is modified. This does not seem like a critical operation that depends on the maintainer of the repository. I wonder if this is intentional, or if it can be changed in the  future.

**Comment by jd** [comment]
Hi @nixpanic!

This is intentional. For now, Mergify completely ignores any pull request that touches its configuration file.

We agree that we should probably remove some of the limitations and keep the check on the merge/queue actions.

I added this to our backlog.

**Reply by nixpanic** [reply]
Thanks for the update! I'll keep our mergify rules to set the labels, and will hopefully notice when they magically start to work :mage:

**Reply by jd** [reply]
@nixpanic https://github.com/Mergifyio/mergify-engine/pull/2603 should fix it 😉

**Reply by nixpanic** [reply]
Thanks!
